ledger/ledger-api-client/src/main/scala/com/digitalasset/ledger/client/services/commands/tracker/CommandTracker.scala

```diff
@@ -222,26 +222,15 @@ private[commands] class CommandTracker[Context](maxDeduplicationTime: () => Opti
             s"A command with id $commandId is already being tracked. CommandIds submitted to the CommandTracker must be unique."
           ) with NoStackTrace
         }
-        maxDeduplicationTime() match {
-          case None =>
-            emit(
-              resultOut,
-              submission.map { _ =>
-                val status = GrpcStatus.toProto(ErrorFactories.missingLedgerConfig().getStatus)
-                Left(NotOkResponse(commandId, status))
-              },
-            )
-          case Some(maxDedup) =>
-            val commandTimeout =
-              maxTimeoutFromDeduplicationPeriod(commands.deduplicationPeriod, maxDedup)
-            val trackingData = TrackingData(
-              commandId = commandId,
-              commandTimeout = commandTimeout,
-              context = submission.context,
-            )
-            pendingCommands += commandId -> trackingData
-            ()
-        }
+        val commandTimeout =
+          timeoutFromDeduplicationPeriod(commands.deduplicationPeriod, maximumExpiryTime)
```

Comment:
I feel like naming is a bit confusing. Instead of going towards a more consistent naming we're now using 3 different terms for this: 
- (deduplication period which I assume will eventually be removed in another PR)
- `commandTimeout` which is our target
- newly-introduced `maximumExpiryTime`  which represents the default timeout and also the "ceil" for timeout if extracted from deduplication

Why introduce `expiryTime` here at all?

Reply:
I have renamed this to `maximumCommandTimeout` as we discussed; I think it's better.